src/containers/ProjectDashboardContainer.jsx

```diff
@@ -20,18 +22,37 @@ const propTypes = {
 };
 
 class ProjectDashboardContainer extends Component {
-
+  constructor() {
+    super();
+    this.handleSelect = this.handleSelect.bind(this);
+    this.state = {
+      translationLanguage: '',
+    };
+  }
   componentDidMount() {
     const { actions } = this.props;
     actions.fetchProject(this.props.params.project_id);
   }
 
+  handleSelect(event) {
+    this.setState({
+      translationLanguage: event.target.textContent,
+    });
+  }
+
   render() {
     const project = this.props.project.data;
     const { fieldguides, pages, workflows, tutorials } = this.props.project;
+    const renderSelectItems = languages.map(item => item.name);
```

Comment:
I think to work with Grommet's Select component that the options prop needs to either be an array of strings or an array of objects where the object is formatted as `{value: string, label: string}`. This line is returning an array of objects that are `{name: string}`. I think you also to set the `onChange` prop for the Select component to use `handleSelect`